plugins/lsp/conform-nvim.nix

```diff
@@ -1,21 +1,31 @@
 {
   lib,
-  helpers,
   config,
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.conform-nvim;
+  inherit (lib) types;
+  inherit (lib.nixvim) defaultNullOpts;
 in
-{
-  options.plugins.conform-nvim = helpers.neovim-plugin.extraOptionsOptions // {
-    enable = mkEnableOption "conform-nvim";
+lib.nixvim.neovim-plugin.mkNeovimPlugin config {
+  name = "conform-nvim";
+  originalName = "conform.nvim";
+  defaultPackage = pkgs.vimPlugins.conform-nvim;
 
-    package = helpers.mkPluginPackageOption "conform-nvim" pkgs.vimPlugins.conform-nvim;
+  # TODO: added 2024-08-23 remove after 24.11
+  deprecateExtraOptions = true;
+  optionsRenamedToSettings = [
```

Comment:
`optionsRenamedToSettings` will converts the rename aliases to snake_case, however looking at your `settingsOptions` and test-cases they should still still be using camelCase.

If so, you'll have to use `mkRenameOptionModule` manually.

Reply:
Sorry, I haven't gotten around to finishing this. I was just posting it to link to the issue and show it was being worked on.